### ISSUE TITLE
Group chats do not call notify() to generate a platform specific notification.

### ISSUE THREAD

**Issue by fun4jimmy** [uTox/uTox#272]
It's very easy to miss group chat messages as there are never any notifications.

It looks like this is because compared to the [FRIEND_MESSAGE event handling](https://github.com/GrayHatter/uTox/blob/cea80f010eca32f0898b87e2e0efd65364db2018/src/tox.c#L1509) code, which calls [friend_addmessage](https://github.com/GrayHatter/uTox/blob/b80ace04297595198bb3f2865e75dbc069433e3b/src/friend.c#L242), the [GROUP_MESSAGE event handling code](https://github.com/GrayHatter/uTox/blob/cea80f010eca32f0898b87e2e0efd65364db2018/src/tox.c#L1627) is missing the section of code that also calls notify shown below.

``` cpp
/* if msg_type is text/action ? create tray popup */
switch(msg->msg_type) {
case MSG_TYPE_TEXT:
case MSG_TYPE_ACTION_TEXT: {
    char_t m[msg->length + 1];
    memcpy(m, msg->msg, msg->length);
    m[msg->length] = 0;
    notify(f->name, f->name_length, m, msg->length, f);
    break;
}
}
```
##

**Comment by tsudoko** [comment]
Group chat notifications should be optional, possible options could be off/messages directed to the user/all messages.

**Comment by fun4jimmy** [comment]
This is still an issue in 0.9.3 but I think after the recent changes to groups the code I originally posted is no longer relevant.

**Comment by endoffile78** [comment]
Fixed in #378
